### PR TITLE
Remove snyk setting - Closes #3252

### LINKED ISSUE #3252: Remove snyk  integration in favour of npm audit
### Description
Since github and npm already supports the snyk feature, we are removing sync from all the prodcuts.

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?
Snyk was used but now npm audit covers the functionality.

### How did I fix it?
Remove snyk settings

### Review checklist

* The PR resolves #3252 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: framework/package.json [hunk 1]
```diff
@@ -137,18 +137,11 @@
...
 		"stampit": "4.2.0",
 		"supertest": "3.3.0"
 	},
 	"lisk": {
 		"minVersion": "1.1.0-rc.0",
 		"protocolVersion": "1.0"
-	},
-	"husky": {
-		"hooks": {
-			"pre-commit": "lint-staged",
```
Comment (limiaspasdaniel) [inline framework/package.json]: Why remove these hooks?

File: lisk/package.json [hunk 2]
```diff
@@ -74,18 +74,11 @@
...
 		"stampit": "4.2.0",
 		"supertest": "3.3.0"
 	},
 	"lisk": {
 		"minVersion": "1.1.0-rc.0",
 		"protocolVersion": "1.0"
-	},
-	"husky": {
-		"hooks": {
-			"pre-commit": "lint-staged",
```
Comment (limiaspasdaniel) [inline lisk/package.json]: Same here

Comment (shuse2) [reply lisk/package.json]: it was `check:dependencies` that was running the snyk, but it doesn't exists anymore

File: framework/package.json (hunk 1 above)
Comment (nazarhussain) [reply framework/package.json]: This line was running the linting commands from https://github.com/okonet/lint-staged. The dependency check was in `pre-push`

```
			"pre-push": "eslint $(git ls-files '*.js') && npm run check:dependencies"
```

Comment (shuse2) [reply framework/package.json]: yes, but hooks are managed in the root level now, and not required here anymore

**Pull request merged**